.github/workflows/build-lint.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
 
   lint:
     needs: build
-    uses: scientist-softserv/actions/.github/workflows/lint.yaml@main
+    uses: scientist-softserv/actions/.github/workflows/lint.yaml@v0.0.3
```

Comment:
I think we want to point to the arm-y_of_one branch here

Reply:
This is the lint step, I think the arm-y_of_one branch only changes things related to the build step: 

https://github.com/scientist-softserv/actions/pull/14/files

The build step currently already points at that branch, which is why there's no diff for it.